Classes/Hooks/WizardItemsHook.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace CPSIT\Multicolumn\Hooks;
+
+use TYPO3\CMS\Backend\Wizard\NewContentElementWizardHookInterface;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class WizardItemsHook implements NewContentElementWizardHookInterface
+{
+
+    /**
+     * modifies WizardItems array
+     *
+     * @param array $wizardItems array of Wizard Items
+     * @param \TYPO3\CMS\Backend\Controller\ContentElement\NewContentElementController $parentObject New Content element wizard
+     *
+     * @return    void
+     */
+    public function manipulateWizardItems(&$wizardItems, &$parentObject)
+    {
+        $multiColumnParentId = (int)GeneralUtility::_GP('tx_multicolumn_parentid');
+        if (empty($multiColumnParentId)) {
+            return;
+        }
+
+        foreach ($wizardItems as &$wizardItem) {
```

Comment:
This still isn't working. Please fix to create a corrent link using defVals[tt_content] for params attribute.

Please test end ensure working functionality before pushing next time.